src/domain/event/EventUtils.ts

```diff
@@ -281,17 +291,35 @@ const getOfferInfoUrl = (
   return getLocalisedString(offer?.infoUrl, locale);
 };
 
+export const getEventAudience = (
```

Comment:
this is quite complex method but not covered with unit tests?

Reply:
combined with the other function and improved tests to have the casing tested